lib/server.js

```diff
@@ -4,11 +4,23 @@ var defaultConfig = require('./config');
 var q = require('q');
 var _ = require('lodash');
 var debug = require('./utils/logger')(__filename);
+var CONSTANTS = require('./constants');
+
+var semver = require('semver');
 
 function initSync(mediator, mbaasApi, datasetId, syncOptions) {
   syncOptions = syncOptions || defaultConfig.syncOptions;
   debug('Sync init');
-  var dataListHandler = function(datasetId, queryParams, cb) {
+  //start the sync service
+  var deferred = q.defer();
+
+  //Checking the version of fh-mbaas-api passed.
+  if (!semver.valid(mbaasApi.getVersion(), CONSTANTS.FH_MBAAS_API_VERSION_RANGE)) {
```

Comment:
This is an API in fh-mbaas-api. The package.json comes from there https://github.com/feedhenry/fh-mbaas-api/blob/master/lib/api.js#L33